projects/rhaiis/workflows/steps/deploy.py

```
@@ -0,0 +1,512 @@
+                    ],
+                    capture_output=True,
+                    text=True,
+                    timeout=15,
+                )
+
+                if result.returncode == 0 and result.stdout.strip() == "True":
+                    elapsed = time.monotonic() - start_time
+                    logger.info(f"InferenceService ready in {elapsed:.1f}s")
+
+                    # Get the service URL
+                    url_result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)